README.md

```diff
@@ -247,20 +247,22 @@ in the `wsk` object. eg:
 }
 ```
 
+The config name changed to `hlx` in version `12.3`. The previous name `wsk` is still supported, but deprecated.
```

Comment:
Maybe also add something here that says that you must use ONE or the OTHER, i.e.

```
{
    "wsk" : {
        "name": "foo"
    },
    "hlx" : {
        "somethingelse": "bar"
    },
}
```
is not supported (`wsk` will be ignored IIUC in that case).

And maybe that should result in a fatal error.

(this and my other comment are potential improvements but not impediments to merging IMO... the behavior as written is sensible and understandable)

Reply:
I added a note in the readme that `hlx` will be used if both are present.